### PR TITLE
US129280 added Settings button to dashboard 

### PR DESCRIPTION
**Pull request by JikaiLong**
Functional Testing
Button works responsively and redirect to corresponding page
Browsers (Chrome, FF, Edge)
Security, devops, perf: N/A
UX and Docs: N/A
ScreenShot:
image

@Brightspace/draco-vis

### CODE REVIEW

File: pages/settings-view.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import '@brightspace-ui-labs/ou-filter/ou-filter';
+import '@brightspace-ui-labs/pagination/pagination';
+
+import { bodyCompactStyles, bodySmallStyles, bodyStandardStyles, heading3Styles } from '@brightspace-ui/core/components/typography/styles';
+import { css, html } from 'lit-element';
+import { Localizer } from '../locales/localizer';
+import { MobxLitElement } from '@adobe/lit-mobx';
+import { SkeletonMixin } from '@brightspace-ui/core/components/skeleton/skeleton-mixin.js';
+
+class SettingsView extends SkeletonMixin(Localizer(MobxLitElement)) {
```
Comment (bemailloux) [inline pages/settings-view.js:13]: I don't think this class needs to have the SkeletonMixin applied to it, since it should load instantly

**Review by bemailloux — APPROVED** [review]
LGTM!

File: pages/settings-view.js (hunk 1 above)
Comment (JikaiLong) [reply pages/settings-view.js:13]: hum, if the settings config is not loaded from the server directly using this component I guess it is not needed.

Comment (JikaiLong) [reply pages/settings-view.js:13]: I will keep it there for now and wait on some further update on the settings page.

**Comment by Vitalii-Misechko** [comment]
Looks good!

- Could you also add a few unit tests (constructor test and axe test)?
- Could you add a visual diff test for the new page

**Comment by JikaiLong** [comment]
> Looks good!
> 
> * Could you also add a few unit tests (constructor test and axe test)?
> * Could you add a visual diff test for the new page

Both are done.

**Comment by JikaiLong** [comment]
Working on Navigation URL for the settings.

**Review by Vitalii-Misechko — APPROVED** [review]
Well done!

**Pull request merged**